src/platform/platform_winuser.c

```diff
@@ -544,6 +527,98 @@ CxPlatUtf8ToWideChar(
     return HRESULT_FROM_WIN32(Error);
 }
 
+BOOLEAN
+CxPlatProcessorGroupInfo(
+    _In_ LOGICAL_PROCESSOR_RELATIONSHIP Relationship,
+    _Out_writes_bytes_to_opt_(*BufferLength,*BufferLength)
+        PSYSTEM_LOGICAL_PROCESSOR_INFORMATION_EX* Buffer,
+    _Out_ PDWORD BufferLength
+    )
```

Comment:
The SAL annotations need a bit of work to indicate this allocates the memory.

Reply:
Also, annotate the result is success only when TRUE. That way you don't have to set any output variables on failure.

Reply:
Ok, I think I got the SAL fix.